fundamentals/impl.tex

```diff
@@ -454,15 +455,20 @@ \subsubsection{Cycamore}
 \textbf{Entity} & \textbf{Archetype} & \textbf{Functionality} \\
 \hline
 Facility & BatchReactor & A reactor model that handles batch refueling, based on pre-determined recipes of compositions. \\
-Facility & Source & This facility generates material of the composition and commodity type specified as input.  \\
-Facility & Sink & This facility is capable of accepting a finite or infinite quantity of some commodity produced in the simulation. \\
 Facility & EnrichmentFacility & This facility enriches uranium at a specified capacity. \\
+Facility & Fab$^*$ & This facility fabricates fuel material based on separated streams. \\
+Facility & Reactor$^*$ & This facility, based on BatchReactor, requests any number of input fuel types and transmutes them to static compositions upon discharge.\\
```

Comment:
The `Reactor` is replacing the `BatchReactor`, so perhaps we want to remove `BatchReactor` entirely? Just an FYI..